tests/phpunit/tests/rest-api/rest-themes-controller.php

```diff
@@ -555,6 +587,29 @@ public function test_theme_template() {
 		$this->assertSame( 'default', $result[0]['template'] );
 	}
 
+	/**
+	 * @ticket 61021
+	 */
+	public function test_theme_template_uri() {
+		wp_set_current_user( self::$admin_id );
+		$request = new WP_REST_Request( 'GET', self::$themes_route );
+		$request->set_param( 'status', array( 'active', 'inactive' ) );
+
+		$response      = rest_get_server()->dispatch( $request );
+		$result        = $response->get_data();
+		$current_theme = wp_get_theme();
+
+		foreach ( $result as $theme_result ) {
+			$this->assertArrayHasKey( 'template_uri', $theme_result );
+			if ( 'active' === $theme_result['status'] ) {
+				$this->assertSame( get_template_directory_uri(), $theme_result['template_uri'] );
```

Comment:
Thanks for updating! LGTM

The only thing I'd do is add a message as the third argument to the assertSames so that failing tests return something meaningful.

Reply:
Good point 👍

Added in: https://github.com/WordPress/wordpress-develop/pull/6399/commits/23cae0544ccfb156f339a9bf9ad859f2eb027784. Happy to tweak the wording if need be.